src/enum/methodAlias.js

```diff
@@ -0,0 +1,9 @@
+import LOG_LEVEL from './logLevel';
+
+export default {
+  error: LOG_LEVEL.ERROR,
+  warn: LOG_LEVEL.WARN,
+  info: LOG_LEVEL.INFO,
+  debug: LOG_LEVEL.DEBUG,
+  log: LOG_LEVEL.DEBUG,
```

Comment:
I think `LOG_LEVEL.INFO` is a more proper default log level, `LOG_LEVEL.DEBUG` will not be logged unless otherwise specified